addons/lunch/models/lunch_supplier.py

```diff
@@ -6,9 +6,10 @@
 from datetime import datetime, time, timedelta
 from textwrap import dedent
 
-from odoo import api, fields, models
+from odoo import _, api, fields, models
 from odoo.osv import expression
 from odoo.tools import float_round
+from odoo.exceptions import UserError
```

Comment:
very small detail, but this line should be above `from odoo.osv import expression` to be ordered alphabetically (an example is given there: [coding_guidelines.html#imports](https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports))